network-api/networkapi/buyersguide/tests.py

```diff
@@ -1,3 +1,272 @@
-# from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APITestCase
 
-# Create your tests here.
+from networkapi.buyersguide.factory import ProductFactory
+from networkapi.buyersguide.models import RangeVote, BooleanVote
+from django.core.management import call_command
+
+
+class ManagementCommandTest(APITestCase):
+
+    def setUp(self):
+        self.vote_url = reverse('product-vote')
```

Comment:
nit, but: this doesn't need to be here, we can get the `reverse('product-vote') value outside the class, as a constant, and refer to that in both `ManagementCommandTest` and `BuyersGuideVoteTest`